coral-spark/src/main/java/com/linkedin/coral/spark/utils/RelDataTypeToHiveTypeStringConverter.java

```diff
@@ -17,6 +17,24 @@
 
 /**
  * Transforms a RelDataType to a Hive type string such that it is parseable and semantically correct.
+ * Some example of Hive type string for a RelDataType are as follows:
+ *
+ * Example 1:
+ * RelDataType:
+ *   struct(s1: integer, s2: varchar)
+ * Hive Type String:
+ *   struct(s1:int,s2:string)
```

Comment:
Could you modify it to be `struct<s1:int,s2:string>`, you may use `&lt;` and `&gt;` for `<` and `>`.

Reply:
sure, thanks for the suggestion